chapters/data-collection.tex

```diff
@@ -2,19 +2,38 @@
 
 \begin{fullwidth}
 Much of the recent push toward credibility in the social sciences has focused on analytical practices.
-We contest that credible research depends, first and foremost, on the quality of the raw data.
-
-This chapter covers the data acquisition
+However, credible development research often depends, first and foremost, on the quality of the raw data.
+This is because, when you are collecting the data yourself,
+or it is provided only to you through a unique partnership,
+there is no way for others to validate that it actually reflects the field reality
```

Comment:
```suggestion
there is no way for others to validate that it accurately reflects the reality
```
"field" seems very restrictive if we're trying to contemplate admin and big data